src/test/java/uk/gov/hmcts/divorce/caseworker/service/print/AosPackPrinterTest.java

```diff
@@ -91,6 +92,78 @@ void shouldPrintAosPackWithD10ForRespondentIfRequiredDocumentsArePresent() {
         assertThat(print.getLetters().get(1).getDivorceDocument()).isSameAs(doc3.getValue());
     }
 
+    @Test
+    void shouldPrintJudicialSeparationAosPackForRespondent() {
+
+        final ListValue<DivorceDocument> doc2 = ListValue.<DivorceDocument>builder()
```

Comment:
could doc2 and doc3, actually be called doc1 and doc2. Small comment but better names!